sigmastate/src/main/scala/sigmastate/lang/syntax/Core.scala

```diff
@@ -6,21 +6,8 @@ import sigmastate.Values._
 import sigmastate.lang.syntax
 
 trait Core extends syntax.Literals {
```

Comment:
Good time to augment the trait with a ScalaDoc

Reply:
Added docs on top of the latest PR here https://github.com/ScorexFoundation/sigmastate-interpreter/commit/1b09c0fad991b2935742e3a68ae7e890e92d79dd